modules/adlooxRtdProvider.md

```diff
@@ -0,0 +1,88 @@
+# Overview
+
+    Module Name: Adloox RTD Provider
+    Module Type: RTD Provider
+    Maintainer: technique@adloox.com
+
+# Description
+
+RTD provider for adloox.com. Contact adops@adloox.com for information.
+
+In addition to populating the ad server key-value targeting, fetched segments (prefixed with `adl_...`) will also populate [First Party Data](https://docs.prebid.org/features/firstPartyData.html), some examples of the segments as described by the Adloox 'Google Publisher Tag Targeting Guidelines' and where they are placed are:
+
+ * Page/Device segments are placed into `fpd.context.data`, for example:
+     * **`adl_ivt`:** `fpd.context.data.adl_ivt` is a boolean
+     * **`adl_ua_old`:** `fpd.context.data.ua_old` is a boolean
+     * **`adl_ip`:** `fpd.context.data.adl_ip` is a string
+ * AdUnit segments are placed into `AdUnit.fpd.context.data`, for example:
+     * **`adl_{dis,vid,aud}`:** `AdUnit.fpd.context.data.adl_{dis,vid,aud}` is an array of integers
+     * **`adl_atf`:** `AdUnit.fpd.context.data.adl_atf` is a boolean (or `-1` on no measure)
+
+**N.B.** this provider does not offer or utilise any user orientated data
+
+This module adds an HTML `<script>` tag to the page to fetch our JavaScript from `https://p.adlooxtracking.com/gpt/a.js` (~3kiB gzipped) to support his integration.
```

Comment:
his --> this

Reply:
c11a14062fe0a6e04f9b6dc91ed5ee362592d0a4